server.go

```diff
@@ -1021,36 +1021,34 @@ func (sp *serverPeer) pushAddrMsg(addresses []*addrmgr.NetAddress) {
 }
 
 // addBanScore increases the persistent and decaying ban score fields by the
-// values passed as parameters. If the resulting score exceeds half of the ban
-// threshold, a warning is logged including the reason provided. Further, if
-// the score is above the ban threshold, the peer will be banned and
-// disconnected.
+// provided values unless banning is disabled.
+//
+// A warning is logged including the reason provided when the resulting score
+// exceeds half of the ban threshold and, unless the peer is whitelisted, it
+// will be banned and disconnected when the score exceeds the ban threshold.
 func (sp *serverPeer) addBanScore(persistent, transient uint32, reason string) bool {
 	// No warning is logged and no score is calculated if banning is disabled.
 	if cfg.DisableBanning {
 		return false
 	}
-	if sp.isWhitelisted {
-		srvrLog.Debugf("Misbehaving whitelisted peer %s: %s", sp, reason)
-		return false
-	}
 
-	warnThreshold := cfg.BanThreshold >> 1
+	// Nothing to do when the score is not being increased.
 	if transient == 0 && persistent == 0 {
-		// The score is not being increased, but a warning message is still
-		// logged if the score is above the warn threshold.
-		score := sp.banScore.Int()
-		if score > warnThreshold {
-			srvrLog.Warnf("Misbehaving peer %s: %s -- ban score is %d, "+
-				"it was not increased this time", sp, reason, score)
-		}
 		return false
 	}
+
+	warnThreshold := cfg.BanThreshold >> 1
```

Comment:
`cfg.BanThreshold / 2`

Reply:
Changed.  Reads exactly the same to me so I don't even notice it.